### PR TITLE
To do list structure

### PR DESCRIPTION
**Pull request by Ochiengsteven**
- I have created an index.html file and wrote my HTML markup there. I have created an empty To Do List placeholder (<div> element). The index.html file is set as a template using the HTML Webpack Plugin.

- I have created an index.js file and set an array of some simple to do tasks (array of objects). Each task object  contains three keys:

  -description [string].

  -completed [bool].
  -index: [number].

- I have a function to iterate over the tasks array and populate an HTML list item element for each task.

- On page load I have rendered the dynamically created list of tasks in the dedicated placeholder. The list appears in order of the index values for each task.

- I have created a style.css and set rules for the To Do List. CSS is also loaded by Webpack Style/CSS Loader.

### CODE REVIEW

**Review by shella12 — COMMENTED** [review]
Hi @Ochiengsteven, 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] doesn't have PR description. Please make sure that your PR is descriptive by adding a meaningful title and a short summary describing the changes you have made.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    description: 'task 2',
+    done: true,
+    id: 1,
+  },
+  {
+    description: 'task 3',
+    done: false,
+    id: 1,
+  },
+];
```
Comment (Tufahel) [inline src/index.js]: - Though your app is working fine. But there is a bug I can see. There are three items but every item has the same `id`. 
- In the project requirements we can see that: "On page load render the dynamically created list of tasks in the dedicated placeholder. `The list should appear in order of the index values for each task.`"
- `The list should appear in order of the index values for each task` this means `id` should be different for each task.
- [ ] Kindly fix the `id` of every item.

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Ochiengsteven :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Descriptive pull request.
✔️ Linter checks are passing.
✔️ Follow the correct GitHub flow.
✔️ Commit messages are meaningful.
✔️ App is working fine.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

- [ ] ***[OPTIONAL]*** Kindly use `capital letter` for the starting of a commit message as it is professional.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — APPROVED** [review]
Hi @Ochiengsteven , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] All linter checks are passing ✔️ 
- [ ] Good commit messages ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body ✔️ 
- [ ] Project up and running ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ochiengsteven** [comment]
giphy

**Pull request merged**